qiskit/transpiler/passes/layout/vf2_utils.py

```diff
@@ -32,7 +32,7 @@ def build_interaction_graph(dag, strict_direction=True):
     im_graph_node_map = {}
     reverse_im_graph_node_map = {}
 
-    class MultiQEncountered(Exception):
+    class MultiQEncounteredError(Exception):
```

Comment:
Not really an error, it's a signal.

Reply:
Probably should not subclass `Exception`.